airflow/models/taskinstance.py

```diff
@@ -1119,24 +1119,23 @@ def get_triggering_events() -> dict[str, list[AssetEvent | AssetEventPydantic]]:
         "outlet_events": OutletEventAccessors(),
         "ds": ds,
         "ds_nodash": ds_nodash,
-        "execution_date": logical_date,
+        "logical_date": logical_date,
         "expanded_ti_count": expanded_ti_count,
         "inlets": task.inlets,
         "inlet_events": InletEventsAccessors(task.inlets, session=session),
-        "logical_date": logical_date,
         "macros": macros,
         "map_index_template": task.map_index_template,
         "next_ds": get_next_ds(),
```

Comment:
Keys removed from the deprecation dict (in `airflow.utils.context`) should also be removed from this dict.

Reply:
removed them